src/tribler-common/tribler_common/tests/test_logger.py

```diff
@@ -0,0 +1,85 @@
+import logging
+from unittest.mock import MagicMock, Mock, call, patch
+
+from tribler_common.logger import get_logger_config_path, setup_logging
+
+
+@patch('tribler_common.logger.__file__', '/a/b/c/logger.py')
```

Comment:
**Question:** Does this mean that `tribler_common.logger.__file__` will return `'/a/b/c/logger.py'?

Reply:
It will return file path, which depends on machine's directory layout and operation system. In test I just use some abstract path to ensure that it will be handled correctly.

Reply:
👍